tests/stdlib/tunittest.nim

```diff
@@ -120,3 +122,31 @@ suite "bug #5784":
       field: int
     var obj: Obj
     check obj.isNil or obj.field == 0
+
+when defined(testing):
```

Comment:
TIL this define is a thing

Reply:
"when defined(testing): export matchFilter" saves matchFilter being exported all the time.
Unless there is a way for the unit test file to import *everything* from a module.